### PR TITLE
Add transport colors

### PR DESCRIPTION
**Pull request by johannah-pik**
Add more colors to plotstyle.csv for detailed transport variables

### CODE REVIEW

File: inst/extdata/plotstyle.csv [hunk 1]
```diff
@@ -579,3 +579,56 @@ EDGE_SSP1.historical;EDGE_SSP1;#45a3d2;3;
...
+Rail|non-HSR;Rail|non|HSR;#000066;;
+Rail|HSR;Rail|HSR;#66CCFF;;
+Road|Bus;Road|Bus;#FFCC33;;
+Aviation|International;Aviation|International;#FF0000;;
+Small;Small;#b8b4b4;;
+Medium ;Medium ;#848484;;
+Mini;Mini;#d1cfcf;;
+SUV;SUV;#C4A1A1;;
+Large ;Large ;#424242;;
+Van ;Van ;#604343;;
```
[the remark this answers is not in the document]
Comment (johannah-pik) [reply inst/extdata/plotstyle.csv]: Sorry, I was already on vacation and will be absent until May 1. Thank you for the comment, I removed the blanks (they were unintended).

File: inst/extdata/plotstyle.csv [hunk 2]
```diff
@@ -579,3 +579,56 @@ EDGE_SSP1.historical;EDGE_SSP1;#45a3d2;3;
...
+Small Cars;Small Cars;#87cefa;;
+Ships international;Ships (international);#cd2626;;
+Ships domestic;Ships (domestic);#B45F04;;
+Passenger Trains;Passenger Trains;#2e8b57;;
+Trucks;Trucks;#ff6a6a;;
+ICE;ICE;#8c8c8c;;
+NG;NG;#d11141;;
+Trucks (<3.5t);Trucks (<3.5t);#ff7f50;;
+Trucks (3.5t-16);Trucks (3.5t-16);#8b0000;;
+Trucks (>16) ;Trucks (>16) ;#fa8072;;
```
Comment (chroetz) [inline inst/extdata/plotstyle.csv:605]: Here is another space before `;`.  I missed that one last time, sorry.

**Review by chroetz — COMMENTED** [review]
After fixing the last space, you can merge it, I think

File: inst/extdata/plotstyle.csv (hunk 2 above)
Comment (Loisel) [inline inst/extdata/plotstyle.csv:605]:
```suggestion
Trucks (>16);Trucks (>16);#fa8072;;
```

**Review by Loisel — COMMENTED** [review]
Please have a look at the comments.

**Pull request merged**